data/fields/fire_mains.json

```
@@ -0,0 +1,15 @@
+{
+    "key": "fire_mains",
+    "type": "combo",
+    "label": "Type",
+    "strings": {
+        "options": {
+            "dry": "Dry Riser Inlet",
+            "wet": "Charged Riser",
+            "yes": "yes",
+            "no": "no"
```

Thinking about it… I believe that the terms dry/wet are probably too domain-specific for a general OSM mapper audience. I think it would be better to describe it like this: 

```suggestion
            "dry": "Dedicated Network (Normally Dry)",
            "wet": "Connected to Mains (Permanently Filled)",
            "yes": "Unknown Type",
            "no": "No"
```